text-run/command-description.js

```diff
@@ -0,0 +1,51 @@
+const child_process = require('child_process')
+const diff = require('jsdiff-console')
+const getCommand = require('./helpers/get-command.js')
+
+module.exports = async function (activity) {
+  const mdDesc = getMd(activity)
+  const cliDesc = getCliDesc(activity)
+  diff(mdDesc, cliDesc)
```

Comment:
Not blocking this PR, but curious if you can use chai here? If you need to format chai errors you can use https://www.npmjs.com/package/assertion-error-formatter. Thats what I setup on cucumber-js

Reply:
Originally chai didn't give me detailed enough diffs of strings or objects. Hence I wrote https://github.com/kevgo/jsdiff-console, which wraps https://github.com/kpdecker/jsdiff. I'll check out assertion-error-formatter to see if it allows doing the same more elegantly. Thanks for the tip!